src/tensor_ops/accurate_gelu/mod.rs

```diff
@@ -0,0 +1,75 @@
+mod cpu_kernel;
+
+#[cfg(feature = "cuda")]
+mod cuda_kernel;
+
+use super::ops::{try_unary_op, UnaryKernel};
+use crate::{shapes::*, tensor::*};
+
+#[repr(C)]
+#[derive(Debug, Default, Copy, Clone)]
+pub struct AccurateGeLUKernelOp;
+
+/// [Accurate Gaussian Linear Unit (GeLU)](https://paperswithcode.com/method/gelu). This is defined as `x * Phi(x)` where `Phi(x)` is the cumulative
+/// distribution function of a standard normal distribution. This can be calculated via the Error
+/// Function `erf(x)` using
+/// ```text
+/// 0.5 * x * (1.0 + erf(x / 2.0.sqrt()))
+/// ```
+/// As an accurate error function is [computationally expensive](https://en.wikipedia.org/wiki/Error_function#Numerical_approximations) it is
+/// possible to approximate the Gaussian Linear Unit with a hyperbolic tangent function `tanh`
+///
+/// ```text
+/// GeLU(x) ~ 0.5 ∗ x ∗ (1.0 + tanh((sqrt(2.0/π) ∗ (x + 0.044715 ∗ x^3)))
+/// ```
+///
+/// See [gelu](crate::tensor_ops::gelu::gelu) to use this approximation
+///
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// # let dev: Cpu = Default::default();
+/// let t = dev.tensor([-1.0, 0.0, 1.0, 2.0]);
+/// let r = t.accurate_gelu();
+/// ```
+pub fn accurate_gelu<S: Shape, E: Dtype, D: UnaryKernel<AccurateGeLUKernelOp, E>, T: Tape<E, D>>(
```

Comment:
Thoughts on calling this `true_gelu`? I wonder if we should rename the other one to `approx_gelu`.

Reply:
How should we deal with breaking name changes like this? A good approach may be to make aliases to the old naming marked as deprecated that direct users to the new names, at least for a major version or two.

Reply:
Yeah that'd be a good way forward if we do want to rename it.

Reply:
I have to admit I like `fast` vs. `accurate`, because `true` and `approx` gelu doesn't really describe the tradeoff: if there is a "true" gelu why would you _not_ choose it? but if there is an accurate and a fast gelu, then I know how to make a decision. What do y'all think? Otherwise fine with the approach of using an alias.

Reply:
I did it now with fast and accurate, but just to emphasize I'm not married to the naming. As repo owners tell me what's the best and I'll use that.

Reply:
Didn't realise type aliased don't work for instantiating zero-size types, so I had to introduce a struct, but the implementation macro is not compatible with the `deprecated` macro, so I just implemented the traits by hand.